cross/toolchain.cmake

```diff
@@ -35,15 +35,6 @@ if(DEFINED ENV{TOOLCHAIN})
   set(TOOLCHAIN $ENV{TOOLCHAIN})
```

Comment:
I am surprised we still use (and have) this file instead of using the shared eng/common/cross/toolchain.cmake. It would be great to start using the shared one (and update it - it is out of date w.r.t. the cross/toolchain.cmake)

Reply:
We can't update the version in `eng/common/cross` until we update all repos that would use it to run on images with CMake 3.14+. I'd prefer to update our `cross/toolchain.cmake` file, then update the images used by the various other repos with native toolsets. Then I'd update the version in `eng/common`. Once that propagates through, I'd remove the local one.